libs/core/src/components/pds-input/stories/pds-input.docs.mdx

```diff
@@ -24,7 +24,7 @@ import * as stories from './pds-input.stories.js';
 
 ### Accessibility
 - The input component comes with a built-in label through the `label` property, it is strongly encouraged that you include it
-- Please provide clear and descriptive error, hint, and placeholder messages
+- Please provide clear and descriptive error, helper message, and placeholder messages
```

Comment:
Should  `helper message` be used as the property name here `helperMessage` or the HTML attribute name `helper-message`, either would suffice?

Reply:
Done